rdkit/Chem/Draw/InteractiveRenderer.py

```diff
@@ -259,7 +259,12 @@ def xmlToNewline(xmlblock):
 
 
 def toDataMol(mol):
-  return "pkl_" + base64.b64encode(mol.ToBinary()).decode("utf-8")
+  return "pkl_" + base64.b64encode(mol.ToBinary(
```

Comment:
Probably the same remark about shortening the property specification applies here

Reply:
Done.